docs/Developers/Writing_Native_Bears.rst

```diff
@@ -411,6 +411,14 @@ strings as a value:
     class SomeBear(Bear):
         LANGUAGES = {'C', 'CPP','C#', 'D'}
 
+.. important::
+
+    - `for language naming always consider the format that is used in the corresponding Wikipedia page, (exception: use CPP instead of cpp or c++);`
+    - `a language definition should be added while writing new bear;`
+    - `the alias used in LANGUAGES should be one of the names in the coala definition.`
+
+
+
```

Comment:
Why so many blanklines?

Reply:
@budukhyash, you need to correct this as well.